templates/patient/card/card_base.html.twig

```diff
@@ -14,6 +14,7 @@
                 {% endif %}
                 {# Display the Edit button #}
                 <a class="{{ btnClass|attr }}" href="{{ linkMethod == "html" ? btnLink : "#" }}" {{ (linkMethod == "html" and requireRestore == true) ? "onclick=\"top.restoreSession()\"" : ""}} {{ linkMethod == "javascript" ? "onclick=\"" ~ btnLink ~ "\"" : ''}}><i class="fa {{ btnIcon }} fa-sm">&nbsp;</i></a>
+                {% block weno %}{% endblock %}
```

Comment:
We shouldn't define special blocks in this template. This should be content and module agnostic to remain as flexible as possible.